zerver/openapi/zulip.yaml

```diff
@@ -19088,6 +19136,14 @@ components:
             was named `can_remove_subscribers_group_id`.
 
             New in Zulip 6.0 (feature level 142).
+        archived:
+          type: boolean
+          description: |
+            A boolean indicating whether the stream is deactivated;
+            true if the stream is deactivated, false otherwise.
+
+            **Changes**: Changed in Zulip 9.0 (feature level 240).
+            Previously, subscriptions only included active streams.
```

Comment:
So this description is a bit weird -- am I correct that this value is always `false`?

Reply:
> am I correct that this value is always false, because we remove all subscribers as part of archiving a stream?

You are correct about that but the problem is that `unsubscribed` use the same schema as `Subscriptions`. So its not necessary that the value will always be false.
```
unsubscribed:
                        type: array
                        items:
                          $ref: "#/components/schemas/Subscriptions"
                        description: |
                          Present if `subscription` is present in `fetch_event_types`.
```

Reply:
OK I think that's a good reason for having the field be always present on subscribed objects, thanks for tracking that down!